PWGJE/Tasks/emcclustermonitor.cxx

```diff
@@ -238,12 +243,21 @@ struct ClusterMonitor {
       auto cellsofcluster = emccluscells.sliceBy(perCluster, cluster.globalIndex());
       double maxamp = 0;
       double ampfraction = 0;
+      std::vector<float> vCellTime;
```

Comment:
is it efficient to create a new vector object for each cluster object? probably better to create it once and then clear for each cluster?

Reply:
Done